README.md

```diff
@@ -41,7 +42,7 @@ requests, code review feedback, and also pull requests.
 ## Documentation
 
 * [Setup Guide](https://firebase.google.com/docs/admin/setup/)
-* [API Reference](https://firebase.google.com/docs/reference/admin/go/)
+* [API Reference](http://godoc.org/github.com/firebase/firebase-admin-go)
```

Comment:
This should point to https://godoc.org/firebase.google.com/go. Both here and above.